README.md

```diff
@@ -203,9 +203,8 @@ Also look on the department’s [page](https://web.cs.toronto.edu/undergraduate/
 
 #### Which Math Course Should I Take?
 
-![math](https://media.giphy.com/media/dJ4vNQ7r72pb4nDhN5/giphy.gif)
```

Comment:
The Gifs still render on github pages fine. Otherwise I'll add then as gifs on the repo to be linked to.
![image](https://user-images.githubusercontent.com/12415411/103241808-5644d780-4922-11eb-9347-f164278ed41b.png)

Reply:
oh, I didn't realize, added them back